providers/trino/src/airflow/providers/trino/hooks/trino.py

```diff
@@ -146,14 +146,14 @@ def __init__(self, *args, **kwargs):
 
     def get_conn(self) -> Connection:
         """Return a connection object."""
-        db = self.get_connection(self.trino_conn_id)  # type: ignore[attr-defined]
+        db = self.get_connection(self.trino_conn_id)
```

Comment:
@gopidesupavan here it is the same like in https://github.com/apache/airflow/pull/53252#discussion_r2202912638 :-(